source/prerequisites.rst

```diff
@@ -77,6 +77,30 @@ Here is a configuration example for ``Nginx`` web server using ``php-fpm``.
         }
     }
 
+IIS configuration
+^^^^^^^^^^^^^^^^^
+
+Here is a configuration exemple for Microsoft IIS.
+
+.. code-block:: nginx
+
+   <?xml version="1.0" encoding="UTF-8"?>
+   <configuration>
+       <system.webServer>
+           <rewrite>
+               <rules>
+                   <rule name="Rewrite to index.php">
```

Comment:
We should `stopProcessing="true"` to ensure other rules that may be added by admins would not be evaluated.

```suggestion
                   <rule name="Rewrite to GLPI" stopProcessing="true">
```

Reply:
ping @zoriax do you agree?

Reply:
Yes sure, I totally agree